package.json

```diff
@@ -22,7 +22,7 @@
   "license": "MPL-2.0",
   "repository": "mozilla/notes",
   "scripts": {
```

Comment:
Since as you found we can do `WEB_EXT_PREF=general.useragent.locale=fr-FR`, should I just add that to the readme?